hms-app/src/components/hackathon-details/ParticipantDetails.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+import { User } from '../../common/types'
+import { Button, Card, createStyles, Group, Modal, Text } from '@mantine/core'
+import { deleteParticipant } from '../../actions/ParticipantActions'
+
+type IProps = {
```

Comment:
so the downside of defining these here and keeping them internal (and giving it the same name as all the other IProps) is that you cannot within the code validate that you are passing in correct values.  I think TS will maybe figure it out for you, but it could be done by the programmer a bit easier if these were exported as something like `ParticipantDetailsProps` and then they would have to cast their input to this component to it.  It should help catch errors sooner